src/components/dropdown/dropdown.js

```diff
@@ -648,6 +652,19 @@ class Dropdown extends React.Component {
 
     return results;
   }
+  /**
+   * positions the portal listBlock in relation to the input.
+   *
+   * @method calculatePosition
+   * @return {void}
+   */
+  calculatePosition() {
+    this.listBlock.style.top = `${this._input.getBoundingClientRect().y
+                                    + (this._input.getBoundingClientRect().height)
+                                    + window.scrollY}px`;
+    this.listBlock.style.width = `${this._input.getBoundingClientRect().width}px`;
+    this.listBlock.style.left = `${this._input.getBoundingClientRect().x}px`;
+  }
```

Comment:
We've got 4 calls to `getBoundingClientRect()` in 3 lines here, and 3 calls to `this.listBlock.style`. It'd be better to call each of these just the once:

``` js
const boundingRect = this._input.getBoundingClientRect();
const top = `${boundingRect.y + boundingRect.height + window.scrollY}px`;
const width = `${boundingRect.width}px`;
const left = `${boundingRect.x}px`;
this.listBlock.setAttribute('style', `left: ${left}; top: ${top}; width: ${width};`);
```

Reply:
Of course